ci/conda_env_cpp.txt

```diff
@@ -22,7 +22,7 @@ azure-storage-blobs-cpp>=12.10.0
 azure-storage-common-cpp>=12.5.0
 azure-storage-files-datalake-cpp>=12.9.0
 benchmark>=1.6.0,!=1.8.4
-boost-cpp>=1.68.0
+libboost-devel>=1.68.0
```

Comment:
It seems that `libboost-devel` is available since 1.82.0: https://anaconda.org/conda-forge/libboost-devel/files?page=3

Can we remove the `>=1.68.0` part?